app/assets/stylesheets/sprangular/_base.scss

```diff
@@ -0,0 +1,284 @@
+body {
+  padding: 70px 0 0 0;
+}
+
+.row.no-padding {
+  [class*="col-"] {
+    padding-left: 0 !important;
+    padding-right: 0 !important;
+  }
+}
+
+.img-responsive {
+  width: 100%;
+}
+
+#loading {
+  position: fixed;
+  z-index: 999;
+  height: 4em;
+  width: 100%;
+  overflow: show;
+  margin: auto;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  text-align: center;
+
+  .fa {
+    color: $body-bg;
+  }
+
+  &:before {
+    content: '';
+    display: block;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba($gray,0.4);
+  }
+}
+
+.navbar-brand img {
+  max-height: 25px;
+}
+
+a {
+  cursor: pointer;
+}
+
+.btn.add-to-cart {
+  span {
+    display: inline-block;
+  }
+
+  .caption {
+    margin-right: 5px;
+    font-weight: bold;
+    border-right: solid 2px rgba($body-bg, 0.4);
+    padding-right: 7px;
+  }
+
+  &.btn-default {
+    .caption {
+      border-right-color: rgba($gray-dark, 0.5);
+    }
+  }
+}
+
+.input-group {
+  .input-group-btn .btn .glyphicon {
+    line-height: 20px;
+  }
+}
+
+#signin {
+  nav a {
+    display: block;
+  }
+}
+
+#initializing {
+  text-align: center;
+}
+
+.aside {
+  width: 100%;
+}
+
+td {
+  &.number {
+    text-align: right;
+  }
+
+  .total & {
+    font-weight: bold;
+  }
+}
+
+footer {
+  background: $gray;
+  color: $gray-light;
+  padding: 10px 5px 5px;
+
+  a {
+    color: $gray-light;
+    font-weight: bold;
+
+    &:hover {
+      color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.